src/3box.js

```diff
@@ -723,6 +725,25 @@ class Box {
   }
 }
 
+function initIPFSRepo () {
+  const sessionID = utils.randInt(10000)
+  const ipfsRootPath = '/ipfs/root/' + sessionID
+
+  const levelInstance = new LevelStore(ipfsRootPath)
+  function randLevelStore () { return levelInstance }
+
+  const repo = new IPFSRepo('/ipfs', {
+    storageBackends: {
+      root: randLevelStore
```

Comment:
If we have a random repo on each start wouldn't the data not be availible? I.e. we get the head from the orbit cache, then we would get the rest of the data from ipfs, but since we are using a new repo the entries would not be available? Am I missing something? Or would it be possible to only randomize the ipfs keys?

Reply:
yeah this is just the root portion of the repo, which includes config info and in the case of ipfsjs still includes the keys/id. The rest of the repo still persists, so things likes blocks (all the data) 

but maybe there is a way to just create a  new id/keys instead of to the same effect creating a new folder/deleting every time

Reply:
Aha, makes sense!